soccer/planning/RRTPlanner.cpp

```diff
@@ -195,109 +212,349 @@ float getTime(InterpolatedPath& path, int index,
         motionConstraints.maxAcceleration, startSpeed, endSpeed);
 }
 
-std::unique_ptr<Planning::InterpolatedPath> RRTPlanner::generateVelocityPath(
-    std::vector<Geometry2d::Point>& points,
+std::unique_ptr<InterpolatedPath> RRTPlanner::generatePath(
```

Comment:
Because encapsulation. The methodology for the method isn't specified. Also not sure about that implementation yet. Not sure if it should optimize and RRT plan. Its fine.